front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -1,19 +1,21 @@
 import React, { memo, useEffect, useState } from 'react';
-import { IOpenPostPageModalEvent } from 'containers/App';
 import { isNilOrError } from 'utils/helperUtils';
 
 // components
 import CloseIconButton from 'components/UI/CloseIconButton';
 import { Icon, useWindowSize } from '@citizenlab/cl2-component-library';
 
 // events
-import eventEmitter from 'utils/eventEmitter';
-import { setIdeaMapCardSelected } from './events';
 import {
   setLeafletMapHoveredMarker,
   leafletMapHoveredMarker$,
+  setLeafletMapSelectedMarker,
 } from 'components/UI/LeafletMap/events';
 
+// router
+import clHistory from 'utils/cl-router/history';
+import { updateSearchParams } from 'utils/cl-router/updateSearchParams';
```

Comment:
These new utilities are nice. Could you add them to the router docs + explain them during the next CDM so that the team knows they are there.